packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +925,124 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting.
+   * This opens a meeting in a new window for the desktop app.
+   * In case of a web app, it will close the current app and open the meeting in the same tab.
```

Comment:
What is the expected behavior on mobile platforms?

Reply:
meeting.joinMeeting is not supported for mobile platform.

Reply:
That seems like important information to document here, rather than leaving developers guessing for the behavior on other platforms.